qdrant-landing/content/documentation/qdrant-cloud-api.md

```diff
@@ -0,0 +1,52 @@
+
+| REST API      | Documentation                                                                        |
+| -------- | ------------------------------------------------------------------------------------ |
+| v.0.1.0 | [OpenAPI Specification](https://cloud.qdrant.io/pa/v1/docs/)                       |
+
+**Note:** This is not the Qdrant REST API. For core product APIs & SDKs, see our list of [interfaces](/documentation/interfaces/)
+
+## Authentication: Connecting to Cloud API
+To interact with the Qdrant Cloud API, you must authenticate using an API key. Each request to the API must include the API key in the **Authorization** header. The API key acts as a bearer token and grants access to your account’s resources.
+
+You can create a Cloud API key in the Cloud Console UI. Go to **Access Management** > **Qdrant Cloud API Keys**.
+![Authentication](/documentation/cloud/authentication.png)
+
+**Note:** Ensure that the API key is kept secure and not exposed in public repositories or logs.  Once authenticated, the API allows you to manage clusters, collections, and perform other operations available to your account.
+
+## Sample API Request
+
+Here's an example of a basic request to **list all clusters** in your Qdrant Cloud account:
+
+```bash
+curl -X 'GET' \
+  'https://cloud.qdrant.io/pa/v1/accounts/<YOUR_ACCOUNT_ID>/clusters' \
+  -H 'accept: application/json' \
+  -H 'Authorization: <YOUR_API_KEY>'
+```
+
+This request will return a list of clusters associated with your account in JSON format.
+
+## Cluster Management  
+In addition to basic CRUD operations, the API supports fine-grained control over cluster resources (CPU, RAM, disk), node configurations, tolerations, and other operational characteristics.
+   - **Get Cluster by ID**: Retrieve detailed information about a specific cluster using the cluster ID and associated account ID.
+   - **Delete Cluster**: Remove a cluster, with optional deletion of backups.
+   - **Update Cluster**: Apply modifications to a cluster's configuration.
+   - **List Clusters**: Get all clusters associated with a specific account, filtered by region or other criteria.
+   - **Create Cluster**: Add new clusters to the account with configurable parameters such as nodes, cloud provider, and regions.
+
+## Cluster Authentication Management
+Use these endpoints to manage your cluster API keys.
+   - **List API Keys**: Retrieve all API keys associated with an account.
+   - **Create API Key**: Generate a new API key for programmatic access.
+   - **Delete API Key**: Revoke access by deleting a specific API key.
+   - **Update API Key**: Modify attributes of an existing API key.
+
+## Booking Packages
+You can manage available service packages across various cloud providers (AWS, GCP, Azure) and their respective regions. Use these endpoints to customize your deployments based on specific needs. 
```

Comment:
I don't understand this, and I'd assume also customers would struggle with this:

* What is a booking package
* What is a deployment in this context
* Why is this paragraph needed at all

I assume with "deployment" you mean a qdrant database cluster. And with "booking package" the configuration of this cluster.

I would add this then in simpler words to the Cluster Management section see above